configure

```diff
@@ -121,6 +121,7 @@ AWKCOMMAND='
 		                 "'$os'" != "CYGWIN" && "'$os'" != "MSVC") { next; }
 		if ($0 == "MSVC"        && "'$os'" != "MSVC")              { next; }
 		if ($0 == "DIRECTMUSIC" && "'$with_direct_music'" == "0")  { next; }
+		if ($0 == "LIBTIMIDITY" && "'$with_libtimidity'" == "0" )  { next; }
```

Comment:
The remove commit had this line:
> if ($0 == "LIBTIMIDITY" && "'$libtimidity'" == "" )        { next; }

Reply:
With original line it doesn't use libtimidity even when detected. Commited fix for detect_pkg_config from TrueBrain.

Reply:
That was still wrong, `"'$libtimidity_config'" == ""` here and unchanged detect_pkg_config should be right.